src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import config.Setting;
+import parser.Parser;
+import processor.Processor;
+import storage.Storage;
+import ui.Ui;
+
```

Comment:
You may want to add the comment header for this class. See [this](https://se-education.org/guides/conventions/java/intermediate.html#comments).